tests/unit/moduleapi/cluster.tcl

```diff
@@ -158,6 +159,16 @@ start_cluster 3 0 [list config_lines $modules] {
         assert_error "ERR Can not execute a command 'set' while the cluster is down" {$node1 do_rm_call set x 1}
     }
 
+    test "Verify Module Auth Succeeds when cluster is down" {
+        r acl setuser foo >pwd on ~* &* +@all
+        # Non Blocking Module Auth
+        assert_equal {OK} [r testmoduleone.rm_register_auth_cb]
```

Comment:
Can you add a pre-condition check like `assert_error {*CLUSTERDOWN*} [r something]` in the test case? It makes it easier to read the test case without reading all the other test cases.

Reply:
Yes. I added a new line to validate that set cmds fails on the "cluster is down" error